# translational 
kp_x: 1
kp_y: 1
kp_z: 50.
#-----
ki_x: 0
ki_y: 0.
ki_z: 0.
#-----
kd_x: 0
kd_y: 0
kd_z: 50.
#-----

# rotational 
kp_phi: 10
kp_theta: 10
kp_psi: .5
#-----
ki_phi: 0.
ki_theta: 0.
ki_psi: 0.
#-----
kd_p: 1
kd_q: 1
kd_r: .5